dali/test/python/test_operator_decoder.py

```diff
@@ -123,14 +125,34 @@ def check(img_type, size, device, threads):
 
 batch_size_alias_test=16
 
+@pipeline_def(batch_size=batch_size_alias_test, device_id=0, num_threads=4)
+def decoder_pipe_multi(device, out_type, files):
+    encoded, _ = fn.readers.file(files=files)
+    decoded = fn.decoders.image(encoded, device=device, output_type=out_type)
+    return decoded
+
+def test_image_decoder_multi():
+    files = glob.glob(os.path.join(test_data_root, "db/single/multichannel/tiff_multichannel") + "/*.tif*")
+    compare_pipelines(decoder_pipe_multi("cpu", out_type=types.ANY_DATA, files=files),
+                      decoder_pipe_multi("mixed", out_type=types.ANY_DATA, files=files),
+                      batch_size=batch_size_alias_test, N_iterations=10,
+                      eps = 1e-03)
+
+def test_image_decoder_multi_fail():
```

Comment:
This fails for CUDA 10 as we don't have nvJPEG2000 there, and we just use host fallback that doesn't apply enforce, but just converts the image to RGB.

Reply:
Fixed.